packages/lists/src/StaticList/StaticList.Example.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { StaticList } from './StaticList';
+
+const ITEMS = new Array(25).fill(0);
+let APPEND_EXAMPLE_ITEMS = [...ITEMS];
+
+function appendItems(): void {
+  APPEND_EXAMPLE_ITEMS = [...APPEND_EXAMPLE_ITEMS, ...new Array(10).fill(0)];
+}
+
+class PerfMark extends React.PureComponent<{ children?: React.ReactNode }> {
```

Comment:
**TODO:** Investigate if this component can be written in a functional way while still capturing timing info from children.

Reply:
**TODO:** Look into using React hooks instead